akka-bench-jmh/src/main/scala/akka/remote/compress/HeavyHittersBenchmark.scala

```diff
@@ -3,55 +3,64 @@
  */
 package akka.remote.compress
 
-import java.util.{PriorityQueue, Random}
-import java.util.concurrent.ThreadLocalRandom
+import java.util.Random
 
 import akka.remote.artery.compress.TopHeavyHitters
 import org.openjdk.jmh.annotations._
+import org.openjdk.jmh.infra.Blackhole
 
+/**
+ * On Macbook pro
+ * [info] Benchmark                                  (n)   Mode  Cnt          Score         Error  Units
+ * [info] HeavyHittersBenchmark.updateHitter        8192  thrpt   40     357405.512 ±    3329.008  ops/s
+ * [info] HeavyHittersBenchmark.updateNotHitter     8192  thrpt   40  259032711.743 ± 7199514.142  ops/s
+ * [info] HeavyHittersBenchmark.updateRandomHitter  8192  thrpt   40    2105102.088 ±   18214.624  ops/s
```

Comment:
Though I think we'll end up maintaining all these tables asynchronously from the incoming messages stream in any case - we'll see soon